users/views.py

```diff
@@ -19,6 +23,54 @@ class UserTokenObtainPairView(TokenObtainPairView):  # type: ignore
     serializer_class = UserTokenObtainPairSerializer
 
 
+class UserFollowView(APIView):
+    def get_object(self, pk: Any) -> Any:
+        try:
+            return User.objects.get(pk=pk)
+        except User.DoesNotExist:
```

Comment:
you can use get_object_or_404

Reply:
I'll use it for the subsequent views

Reply:
Coz they both achieve the same thing but none of them offers any performance advantage over the other. I was comfortable with writing the more verbose method. Also it will require me to rewrite my tests and views files.